src/languages/en.ts

```diff
@@ -1072,6 +1072,14 @@ export default {
             },
         },
     },
+    workflowsDelayedSubmissionPage: {
+        autoReportingErrorMessage: 'The delayed submission parameter could not be changed. Please try again or contact support.',
+        autoReportingFrequencyErrorMessage: 'The submission frequency could not be changed. Please try again or contact support.',
+        monthlyOffsetErrorMessage: 'The monthly frequency could not be changed. Please try again or contact support.',
+    },
+    workflowsApprovalPage: {
+        genericErrorMessage: 'The approver could not be changed. Please try again or contact support.',
```

Comment:
Has marketing reviewed these copies?

Reply:
Asked about it [here](https://expensify.slack.com/archives/C03SSAQG5/p1710837751504119)

Reply:
@lakchote This is usually done using Marketing label autoassigner on the linked issue so we get one dedicated person for this